framework/src/modules/token/constants.ts

```diff
@@ -38,19 +38,20 @@ export const TOKEN_ID_LENGTH = CHAIN_ID_LENGTH + LOCAL_ID_LENGTH;
 export const LOCAL_ID_LSK = Buffer.alloc(LOCAL_ID_LENGTH, 0);
 export const CHAIN_ID_LSK = Buffer.from([0, 0, 0, 0]);
 export const TOKEN_ID_LSK = Buffer.from([0, 0, 0, 0, 0, 0, 0, 0]);
-export const USER_SUBSTORE_INITIALIZATION_FEE = '5000000';
-export const TOKEN_ID_STORE_INITIALIZATION = TOKEN_ID_LSK;
-
+export const USER_SUBSTORE_INITIALIZATION_FEE = BigInt(5000000);
 export const ESCROW_SUBSTORE_INITIALIZATION_FEE = BigInt(5000000);
 
+export const TOKEN_ID_STORE_INITIALIZATION = TOKEN_ID_LSK;
```

Comment:
maybe check if this value is used, if not we can remove them

Reply:
It seems it is used only in 1 single test case in `commands/transfer.spec.ts`, but nowhere in the app code:

_should pass if token balance for token ID TOKEN_ID_STORE_INITIALIZATION is at least the sum of configured initialization fee and transaction amount_

Should I move that constant from `constants.ts` to the test? Or, if it's not used anywhere in the code, should I just remove both the constant and the test case?

Reply:
Removed `TOKEN_ID_STORE_INITIALIZATION` from constants, removed the test case and changed value for `feeTokenID` id `Transfer` test suite to be `TOKEN_ID_LSK`.